### PR TITLE
wmlxgettext and wmlindent change exit code

### PR DESCRIPTION
**Pull request by max-torch**
For keyboard interrupt change exit code from 0 to 1.

### CODE REVIEW

**Comment by Pentarctagon** [comment]
@Elvish-Hunter feel free to click the merge button :P

File: data/tools/wmlindent [hunk 1]
```diff
@@ -306,7 +306,7 @@ def sigint_handler(signal, frame):
     This overrides Python's default behavior of displaying a traceback when ctrl-c is pressed.
     """
     print ('Aborted by pressing ctrl-c')
-    sys.exit(0)
+    sys.exit(1)
```
Comment (soliton-) [inline data/tools/wmlindent:309]: The comments above should stay in sync with the code.

**Review by soliton- — CHANGES_REQUESTED** [review]

File: data/tools/wmlindent (hunk 1 above)
Comment (max-torch) [reply data/tools/wmlindent:309]: > The comments above should stay in sync with the code.

Not sure what you are referring to. Does "comments" refer to the function docstring or to the print message? I don't quite understand the request actually, could you rephrase it?

Comment (soliton-) [reply data/tools/wmlindent:309]: Yes, the docstring spells out what is done... `a one-line message is displayed and Python exits with Status 0`

Comment (max-torch) [reply data/tools/wmlindent:309]: resolved in https://github.com/wesnoth/wesnoth/pull/6500/commits/42faf4f1ab3717e7364efc94b36f93c28fed7dfb

**Review by Elvish-Hunter — APPROVED** [review]
